wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/markup/html/autocomplete/wicket-autocomplete.js

```diff
@@ -484,10 +500,10 @@
 
 		function getWindowScrollXY() {
 			var scrOfX = 0, scrOfY = 0;
-			if( typeof( window.pageYOffset ) === 'number' ) {
+			if( typeof( window.scrollY ) === 'number' ) {
 				//Netscape compliant
-				scrOfY = window.pageYOffset;
-				scrOfX = window.pageXOffset;
+				scrOfY = window.scrollY;
```

Comment:
Did you face problems with `pageXOffset` ?

Reply:
No, it was suggested by the IDE because pageXOffset is deprecated 
https://github.com/microsoft/TypeScript/blob/main/src/lib/dom.generated.d.ts#L25491